config/install/field.field.node.title.field_eye_color.yml

```diff
@@ -0,0 +1,18 @@
+langcode: en
```

Comment:
This file can be removed. It refers to the node type "title" which we need to remove in favor of "people".